
[     UTC     ] Logs for barbeariadonmunhoz.streamlit.app/
────────────────────────────────────────────────────────────────────────────────────────
[14:15:10] 🖥 Provisioning machine...
[14:15:10] 🎛 Preparing system...
[14:15:10] ⛓ Spinning up manager process...
[14:15:06] 🚀 Starting up repository: 'streamlit_academy', branch: 'main', main module: 'Analise_Barbearia/analise_barbearia.py'
[14:15:06] 🐙 Cloning repository...
[14:15:08] 🐙 Cloning into '/mount/src/streamlit_academy'...
Warning: Permanently added the ED25519 host key for IP address '140.82.116.3' to the list of known hosts.

[14:15:08] 🐙 Cloned repository!
[14:15:08] 🐙 Pulling code changes from Github...
[14:15:09] 📦 Processing dependencies...
Check if streamlit is installed
cat: /mount/admin/install_path: No such file or directory

──────────────────────────────── Installing Streamlit ──────────────────────────────────

Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 37 packages in 394ms
Prepared 37 packages in 1.59s
Installed 37 packages in 87ms
 + altair[2025-08-09 14:15:11.833764] ==5.5.0
 + attrs==25.3.0
 + blinker==1.9.0
 + cachetools==6.1.0
 + certifi==2025.8.3
 + charset-normalizer==3.4.3
 + click==8.2.1
 + gitdb==4.0.12
 + gitpython==3.1.45
 + idna==3.10
 + jinja2==3.1.6
 + jsonschema==4.25.0
 + jsonschema-specifications==2025.4.1
 + markupsafe[2025-08-09 14:15:11.834066] ==3.0.2
 + narwhals==2.0.1
 + numpy==2.3.2
 + packaging==25.0
 + pandas==2.3.1
 + pillow==11.3.0
 + protobuf==6.31.1
 + pyarrow==21.0.0
 +[2025-08-09 14:15:11.834302]  pydeck==0.9.1
 + python-dateutil==2.9.0.post0
 + pytz==2025.2
 + referencing==0.36.2
 + requests==2.32.4
 + rpds-py==0.27.0
 [2025-08-09 14:15:11.834493] + six==1.17.0
 + smmap==5.0.2
 + streamlit==1.48.0
 + tenacity==9.1.2
 + toml==0.10.2
 + tornado==6.5.2
 +[2025-08-09 14:15:11.834660]  typing-extensions==4.14.1
 + tzdata==2025.2
 + urllib3==2.5.0
 + watchdog==6.0.0

────────────────────────────────────────────────────────────────────────────────────────

[14:15:12] 📦 Processed dependencies!
cat: /mount/admin/install_path: No such file or directory



2025-08-09 14:15:22.389 Uncaught app execution
Traceback (most recent call last):
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/exec_code.py", line 128, in exec_func_with_error_handling
    result = func()
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 669, in code_to_exec
    exec(code, module.__dict__)  # noqa: S102
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py", line 3, in <module>
    import plotly.express as px
ModuleNotFoundError: No module named 'plotly'
[14:15:06] 🚀 Starting up repository: 'streamlit_academy', branch: 'main', main module: 'Analise_Barbearia/analise_barbearia.py'
[14:15:06] 🐙 Cloning repository...
[14:15:08] 🐙 Cloning into '/mount/src/streamlit_academy'...
Warning: Permanently added the ED25519 host key for IP address '140.82.116.3' to the list of known hosts.

[14:15:08] 🐙 Cloned repository!
[14:15:08] 🐙 Pulling code changes from Github...
[14:15:09] 📦 Processing dependencies...
Check if streamlit is installed
cat: /mount/admin/install_path: No such file or directory

──────────────────────────────── Installing Streamlit ──────────────────────────────────

Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 37 packages in 394ms
Prepared 37 packages in 1.59s
Installed 37 packages in 87ms
 + altair[2025-08-09 14:15:11.833764] ==5.5.0
 + attrs==25.3.0
 + blinker==1.9.0
 + cachetools==6.1.0
 + certifi==2025.8.3
 + charset-normalizer==3.4.3
 + click==8.2.1
 + gitdb==4.0.12
 + gitpython==3.1.45
 + idna==3.10
 + jinja2==3.1.6
 + jsonschema==4.25.0
 + jsonschema-specifications==2025.4.1
 + markupsafe[2025-08-09 14:15:11.834066] ==3.0.2
 + narwhals==2.0.1
 + numpy==2.3.2
 + packaging==25.0
 + pandas==2.3.1
 + pillow==11.3.0
 + protobuf==6.31.1
 + pyarrow==21.0.0
 +[2025-08-09 14:15:11.834302]  pydeck==0.9.1
 + python-dateutil==2.9.0.post0
 + pytz==2025.2
 + referencing==0.36.2
 + requests==2.32.4
 + rpds-py==0.27.0
 [2025-08-09 14:15:11.834493] + six==1.17.0
 + smmap==5.0.2
 + streamlit==1.48.0
 + tenacity==9.1.2
 + toml==0.10.2
 + tornado==6.5.2
 +[2025-08-09 14:15:11.834660]  typing-extensions==4.14.1
 + tzdata==2025.2
 + urllib3==2.5.0
 + watchdog==6.0.0

────────────────────────────────────────────────────────────────────────────────────────

[14:15:12] 📦 Processed dependencies!
cat: /mount/admin/install_path: No such file or directory



2025-08-09 14:15:22.389 Uncaught app execution
Traceback (most recent call last):
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/exec_code.py", line 128, in exec_func_with_error_handling
    result = func()
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 669, in code_to_exec
    exec(code, module.__dict__)  # noqa: S102
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py", line 3, in <module>
    import plotly.express as px
ModuleNotFoundError: No module named 'plotly'
[14:17:21] 🐙 Pulling code changes from Github...
[14:17:22] 📦 Processing dependencies...

──────────────────────────────────────── uv ───────────────────────────────────────────

Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 91 packages in 463ms
Prepared 63 packages in 4.02s
Uninstalled 9 packages in 208ms
Installed 63 packages in 212ms
 + anyio==4.9.0
 + beautifulsoup4==4.13.4
 + bleach==6.2.0
 + bs4==0.0.2
 - certifi==2025.8.3
 + certifi==2025.6.15
 - charset-normalizer==3.4.3
 + charset-normalizer==3.4.2
 + colorama==0.4.6
 + coloredlogs==15.0.1
 + contourpy==1.3.2
 + cycler==0.12.1
 + distro==1.9.0
 + et-xmlfile==2.0.0
 + flatbuffers[2025-08-09 14:17:27.872683] ==25.2.10
 + fonttools==4.58.4
 + h11==0.16.0
 + httpcore==1.0.9
 + httpx==0.28.1
 + humanfriendly==10.0
 + imageio==2.37.0
 + jpype1==1.6.0
 + kaggle==1.7.4.5
 + kagglehub==0.3.12
 + kiwisolver==1.4.8
 + lazy-loader==0.4
 + llvmlite==0.44.0
 + lxml==6.0.0
 + matplotlib[2025-08-09 14:17:27.872921] ==3.10.3
 + mpmath==1.3.0
 + mysql-connector-python==9.4.0
 - narwhals==2.0.1
 + narwhals==1.48.1
 + networkx==3.5
 + numba==0.61.2
 - numpy==2.3.2
 + numpy==2.2.6
 + onnxruntime==1.22.1
 + opencv-python-headless==4.12.0.88
 + openpyxl==3.1.5
 - pandas==2.3.1
 + pandas==2.3.0
 - pillow==11.3.0
 + pillow==11.2.1
 + platformdirs==4.3.8
 + plotly==6.2.0
 + pooch==1.8.2
 + pymatting==1.1.14
 + pyparsing==3.2.3
 + pyreadline3==3.5.4
 + python-bcb==0.3.3
 + python-slugify==8.0.4
 + pyyaml==6.0.2
 + rembg==2.0.67
 - rpds-py==0.27.0
 + rpds-py==0.26.0
 + scikit-image==0.25.2
 + scipy==1.16.1
 + seaborn==0.13.2
 + setuptools==80.9.0
 + sniffio==1.3.1
 + soupsieve==2.7
 - streamlit==1.48.0
 + streamlit==1.47.0
 + sympy==1.14.0
 + tabula-py==2.10.0
 + text-unidecode==1.3
 + tifffile==2025.6.11
 - tornado==6.5.2
 + tornado==6.5.1
 + tqdm==4.67.1
 + webencodings==0.5.1
Checking if Streamlit is installed
Found Streamlit version 1.47.0 in the environment
Installing rich for an improved exception logging
Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 4 packages in 128ms
Prepared 4 packages in 124ms
Installed 4 packages in 12ms
 + markdown-it-py==3.0.0
 + mdurl==0.1.2
 + pygments==2.19.2
 + rich==14.1.0

────────────────────────────────────────────────────────────────────────────────────────

[14:17:31] 🐍 Python dependencies were installed from /mount/src/streamlit_academy/Analise_Barbearia/requirements.txt using uv.
[14:17:31] 📦 Processed dependencies!
  Stopping...
────────────────────── Traceback (most recent call last) ───────────────────────
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:17 in     
  carregar_dados                                                                
                                                                                
     14 │   Função para carregar e transformar os dados da planilha Excel.      
     15 │   """                                                                 
     16 │   try:                                                                
  ❱  17 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     18 │   except FileNotFoundError:                                           
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'

During handling of the above exception, another exception occurred:

────────────────────── Traceback (most recent call last) ───────────────────────
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/exec_code.py:128 in exec_func_with_error_handling                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/script_runner.py:669 in code_to_exec                                     
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:32 in     
  <module>                                                                      
                                                                                
     29 │   return df                                                           
     30                                                                         
     31                                                                         
  ❱  32 df_original = carregar_dados()                                          
     33                                                                         
     34 # --- 3. BARRA LATERAL (FILTROS) ---                                    
     35 st.sidebar.image("logo_DonMunhoz_semFundo.png", use_container_width=Tr  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:227 in __call__                                                
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:269 in _get_or_create_cached_value                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:328 in _handle_cache_miss                                      
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:22 in     
  carregar_dados                                                                
                                                                                
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
     21 │   │   )                                                               
  ❱  22 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     23 │                                                                       
     24 │   df["Valor"] = pd.to_numeric(df["Valor"], errors="coerce")           
     25 │   df["Data"] = pd.to_datetime(df["Data"], errors="coerce")            
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'
[14:17:34] 🔌 Disconnecting...



[14:17:36] 🖥 Provisioning machine...
[14:17:36] 🎛 Preparing system...
[14:17:36] ⛓ Spinning up manager process...
[14:17:34] 🔄 Updated app!
────────────────────── Traceback (most recent call last) ───────────────────────
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:17 in     
  carregar_dados                                                                
                                                                                
     14 │   Função para carregar e transformar os dados da planilha Excel.      
     15 │   """                                                                 
     16 │   try:                                                                
  ❱  17 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     18 │   except FileNotFoundError:                                           
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'

During handling of the above exception, another exception occurred:

────────────────────── Traceback (most recent call last) ───────────────────────
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/exec_code.py:128 in exec_func_with_error_handling                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/script_runner.py:669 in code_to_exec                                     
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:32 in     
  <module>                                                                      
                                                                                
     29 │   return df                                                           
     30                                                                         
     31                                                                         
  ❱  32 df_original = carregar_dados()                                          
     33                                                                         
     34 # --- 3. BARRA LATERAL (FILTROS) ---                                    
     35 st.sidebar.image("logo_DonMunhoz_semFundo.png", use_container_width=Tr  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:225 in __call__                                                
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:267 in _get_or_create_cached_value                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:326 in _handle_cache_miss                                      
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:22 in     
  carregar_dados                                                                
                                                                                
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
     21 │   │   )                                                               
  ❱  22 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     23 │                                                                       
     24 │   df["Valor"] = pd.to_numeric(df["Valor"], errors="coerce")           
     25 │   df["Data"] = pd.to_datetime(df["Data"], errors="coerce")            
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'
[14:15:06] 🚀 Starting up repository: 'streamlit_academy', branch: 'main', main module: 'Analise_Barbearia/analise_barbearia.py'
[14:15:06] 🐙 Cloning repository...
[14:15:08] 🐙 Cloning into '/mount/src/streamlit_academy'...
Warning: Permanently added the ED25519 host key for IP address '140.82.116.3' to the list of known hosts.

[14:15:08] 🐙 Cloned repository!
[14:15:08] 🐙 Pulling code changes from Github...
[14:15:09] 📦 Processing dependencies...
Check if streamlit is installed
cat: /mount/admin/install_path: No such file or directory

──────────────────────────────── Installing Streamlit ──────────────────────────────────

Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 37 packages in 394ms
Prepared 37 packages in 1.59s
Installed 37 packages in 87ms
 + altair[2025-08-09 14:15:11.833764] ==5.5.0
 + attrs==25.3.0
 + blinker==1.9.0
 + cachetools==6.1.0
 + certifi==2025.8.3
 + charset-normalizer==3.4.3
 + click==8.2.1
 + gitdb==4.0.12
 + gitpython==3.1.45
 + idna==3.10
 + jinja2==3.1.6
 + jsonschema==4.25.0
 + jsonschema-specifications==2025.4.1
 + markupsafe[2025-08-09 14:15:11.834066] ==3.0.2
 + narwhals==2.0.1
 + numpy==2.3.2
 + packaging==25.0
 + pandas==2.3.1
 + pillow==11.3.0
 + protobuf==6.31.1
 + pyarrow==21.0.0
 +[2025-08-09 14:15:11.834302]  pydeck==0.9.1
 + python-dateutil==2.9.0.post0
 + pytz==2025.2
 + referencing==0.36.2
 + requests==2.32.4
 + rpds-py==0.27.0
 [2025-08-09 14:15:11.834493] + six==1.17.0
 + smmap==5.0.2
 + streamlit==1.48.0
 + tenacity==9.1.2
 + toml==0.10.2
 + tornado==6.5.2
 +[2025-08-09 14:15:11.834660]  typing-extensions==4.14.1
 + tzdata==2025.2
 + urllib3==2.5.0
 + watchdog==6.0.0

────────────────────────────────────────────────────────────────────────────────────────

[14:15:12] 📦 Processed dependencies!
cat: /mount/admin/install_path: No such file or directory



2025-08-09 14:15:22.389 Uncaught app execution
Traceback (most recent call last):
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/exec_code.py", line 128, in exec_func_with_error_handling
    result = func()
  File "/home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 669, in code_to_exec
    exec(code, module.__dict__)  # noqa: S102
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "/mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py", line 3, in <module>
    import plotly.express as px
ModuleNotFoundError: No module named 'plotly'
[14:17:21] 🐙 Pulling code changes from Github...
[14:17:22] 📦 Processing dependencies...

──────────────────────────────────────── uv ───────────────────────────────────────────

Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 91 packages in 463ms
Prepared 63 packages in 4.02s
Uninstalled 9 packages in 208ms
Installed 63 packages in 212ms
 + anyio==4.9.0
 + beautifulsoup4==4.13.4
 + bleach==6.2.0
 + bs4==0.0.2
 - certifi==2025.8.3
 + certifi==2025.6.15
 - charset-normalizer==3.4.3
 + charset-normalizer==3.4.2
 + colorama==0.4.6
 + coloredlogs==15.0.1
 + contourpy==1.3.2
 + cycler==0.12.1
 + distro==1.9.0
 + et-xmlfile==2.0.0
 + flatbuffers[2025-08-09 14:17:27.872683] ==25.2.10
 + fonttools==4.58.4
 + h11==0.16.0
 + httpcore==1.0.9
 + httpx==0.28.1
 + humanfriendly==10.0
 + imageio==2.37.0
 + jpype1==1.6.0
 + kaggle==1.7.4.5
 + kagglehub==0.3.12
 + kiwisolver==1.4.8
 + lazy-loader==0.4
 + llvmlite==0.44.0
 + lxml==6.0.0
 + matplotlib[2025-08-09 14:17:27.872921] ==3.10.3
 + mpmath==1.3.0
 + mysql-connector-python==9.4.0
 - narwhals==2.0.1
 + narwhals==1.48.1
 + networkx==3.5
 + numba==0.61.2
 - numpy==2.3.2
 + numpy==2.2.6
 + onnxruntime==1.22.1
 + opencv-python-headless==4.12.0.88
 + openpyxl==3.1.5
 - pandas==2.3.1
 + pandas==2.3.0
 - pillow==11.3.0
 + pillow==11.2.1
 + platformdirs==4.3.8
 + plotly==6.2.0
 + pooch==1.8.2
 + pymatting==1.1.14
 + pyparsing==3.2.3
 + pyreadline3==3.5.4
 + python-bcb==0.3.3
 + python-slugify==8.0.4
 + pyyaml==6.0.2
 + rembg==2.0.67
 - rpds-py==0.27.0
 + rpds-py==0.26.0
 + scikit-image==0.25.2
 + scipy==1.16.1
 + seaborn==0.13.2
 + setuptools==80.9.0
 + sniffio==1.3.1
 + soupsieve==2.7
 - streamlit==1.48.0
 + streamlit==1.47.0
 + sympy==1.14.0
 + tabula-py==2.10.0
 + text-unidecode==1.3
 + tifffile==2025.6.11
 - tornado==6.5.2
 + tornado==6.5.1
 + tqdm==4.67.1
 + webencodings==0.5.1
Checking if Streamlit is installed
Found Streamlit version 1.47.0 in the environment
Installing rich for an improved exception logging
Using uv pip install.
Using Python 3.13.5 environment at /home/adminuser/venv
Resolved 4 packages in 128ms
Prepared 4 packages in 124ms
Installed 4 packages in 12ms
 + markdown-it-py==3.0.0
 + mdurl==0.1.2
 + pygments==2.19.2
 + rich==14.1.0

────────────────────────────────────────────────────────────────────────────────────────

[14:17:31] 🐍 Python dependencies were installed from /mount/src/streamlit_academy/Analise_Barbearia/requirements.txt using uv.
[14:17:31] 📦 Processed dependencies!
  Stopping...
────────────────────── Traceback (most recent call last) ───────────────────────
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:17 in     
  carregar_dados                                                                
                                                                                
     14 │   Função para carregar e transformar os dados da planilha Excel.      
     15 │   """                                                                 
     16 │   try:                                                                
  ❱  17 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     18 │   except FileNotFoundError:                                           
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'

During handling of the above exception, another exception occurred:

────────────────────── Traceback (most recent call last) ───────────────────────
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/exec_code.py:128 in exec_func_with_error_handling                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/script_runner.py:669 in code_to_exec                                     
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:32 in     
  <module>                                                                      
                                                                                
     29 │   return df                                                           
     30                                                                         
     31                                                                         
  ❱  32 df_original = carregar_dados()                                          
     33                                                                         
     34 # --- 3. BARRA LATERAL (FILTROS) ---                                    
     35 st.sidebar.image("logo_DonMunhoz_semFundo.png", use_container_width=Tr  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:227 in __call__                                                
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:269 in _get_or_create_cached_value                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:328 in _handle_cache_miss                                      
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:22 in     
  carregar_dados                                                                
                                                                                
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
     21 │   │   )                                                               
  ❱  22 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     23 │                                                                       
     24 │   df["Valor"] = pd.to_numeric(df["Valor"], errors="coerce")           
     25 │   df["Data"] = pd.to_datetime(df["Data"], errors="coerce")            
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'



[14:17:34] 🔄 Updated app!
────────────────────── Traceback (most recent call last) ───────────────────────
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:17 in     
  carregar_dados                                                                
                                                                                
     14 │   Função para carregar e transformar os dados da planilha Excel.      
     15 │   """                                                                 
     16 │   try:                                                                
  ❱  17 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     18 │   except FileNotFoundError:                                           
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'

During handling of the above exception, another exception occurred:

────────────────────── Traceback (most recent call last) ───────────────────────
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/exec_code.py:128 in exec_func_with_error_handling                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/script_runner.py:669 in code_to_exec                                     
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:32 in     
  <module>                                                                      
                                                                                
     29 │   return df                                                           
     30                                                                         
     31                                                                         
  ❱  32 df_original = carregar_dados()                                          
     33                                                                         
     34 # --- 3. BARRA LATERAL (FILTROS) ---                                    
     35 st.sidebar.image("logo_DonMunhoz_semFundo.png", use_container_width=Tr  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:225 in __call__                                                
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:267 in _get_or_create_cached_value                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:326 in _handle_cache_miss                                      
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:22 in     
  carregar_dados                                                                
                                                                                
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
     21 │   │   )                                                               
  ❱  22 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     23 │                                                                       
     24 │   df["Valor"] = pd.to_numeric(df["Valor"], errors="coerce")           
     25 │   df["Data"] = pd.to_datetime(df["Data"], errors="coerce")            
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'
────────────────────── Traceback (most recent call last) ───────────────────────
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:17 in     
  carregar_dados                                                                
                                                                                
     14 │   Função para carregar e transformar os dados da planilha Excel.      
     15 │   """                                                                 
     16 │   try:                                                                
  ❱  17 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     18 │   except FileNotFoundError:                                           
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'

During handling of the above exception, another exception occurred:

────────────────────── Traceback (most recent call last) ───────────────────────
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/exec_code.py:128 in exec_func_with_error_handling                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/scriptru  
  nner/script_runner.py:669 in code_to_exec                                     
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:32 in     
  <module>                                                                      
                                                                                
     29 │   return df                                                           
     30                                                                         
     31                                                                         
  ❱  32 df_original = carregar_dados()                                          
     33                                                                         
     34 # --- 3. BARRA LATERAL (FILTROS) ---                                    
     35 st.sidebar.image("logo_DonMunhoz_semFundo.png", use_container_width=Tr  
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:225 in __call__                                                
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:267 in _get_or_create_cached_value                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/streamlit/runtime/caching/  
  cache_utils.py:326 in _handle_cache_miss                                      
                                                                                
  /mount/src/streamlit_academy/Analise_Barbearia/analise_barbearia.py:22 in     
  carregar_dados                                                                
                                                                                
     19 │   │   st.error(                                                       
     20 │   │   │   "Arquivo 'GERAL_AGENDAMENTOS_reais.xlsx' não encontrado. C  
     21 │   │   )                                                               
  ❱  22 │   │   df = pd.read_excel("GERAL_AGENDAMENTOS_FICTICIOS.xlsx")         
     23 │                                                                       
     24 │   df["Valor"] = pd.to_numeric(df["Valor"], errors="coerce")           
     25 │   df["Data"] = pd.to_datetime(df["Data"], errors="coerce")            
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:4  
  95 in read_excel                                                              
                                                                                
     492 │                                                                      
     493 │   if not isinstance(io, ExcelFile):                                  
     494 │   │   should_close = True                                            
  ❱  495 │   │   io = ExcelFile(                                                
     496 │   │   │   io,                                                        
     497 │   │   │   storage_options=storage_options,                           
     498 │   │   │   engine=engine,                                             
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  550 in __init__                                                               
                                                                                
    1547 │   │   │   if xlrd_version is not None and isinstance(path_or_buffer  
    1548 │   │   │   │   ext = "xls"                                            
    1549 │   │   │   else:                                                      
  ❱ 1550 │   │   │   │   ext = inspect_excel_format(                            
    1551 │   │   │   │   │   content_or_path=path_or_buffer, storage_options=s  
    1552 │   │   │   │   )                                                      
    1553 │   │   │   │   if ext is None:                                        
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/excel/_base.py:1  
  402 in inspect_excel_format                                                   
                                                                                
    1399 │   if isinstance(content_or_path, bytes):                             
    1400 │   │   content_or_path = BytesIO(content_or_path)                     
    1401 │                                                                      
  ❱ 1402 │   with get_handle(                                                   
    1403 │   │   content_or_path, "rb", storage_options=storage_options, is_te  
    1404 │   ) as handle:                                                       
    1405 │   │   stream = handle.handle                                         
                                                                                
  /home/adminuser/venv/lib/python3.13/site-packages/pandas/io/common.py:882 in  
  get_handle                                                                    
                                                                                
     879 │   │   │   )                                                          
     880 │   │   else:                                                          
     881 │   │   │   # Binary mode                                              
  ❱  882 │   │   │   handle = open(handle, ioargs.mode)                         
     883 │   │   handles.append(handle)                                         
     884 │                                                                      
     885 │   # Convert BytesIO or file objects passed with an encoding          
────────────────────────────────────────────────────────────────────────────────
FileNotFoundError: [Errno 2] No such file or directory: 
'GERAL_AGENDAMENTOS_FICTICIOS.xlsx'